crates/core/src/sql/datetime.rs

```diff
@@ -103,6 +103,11 @@ impl Datetime {
 	pub fn to_u64(&self) -> u64 {
 		self.0.timestamp_nanos_opt().unwrap_or_default() as u64
 	}
+
+	/// Convert to nanosecond timestamp.
+	pub fn to_i64(&self) -> i64 {
+		self.0.timestamp_nanos_opt().unwrap_or_default()
```

Comment:
I can see we do the same thing with `to_u64` but I think it's better to return either an option or a result here to give the caller a chance to handle an out of range date.

Reply:
👍

Reply:
Fixed in [464a9b1](https://github.com/surrealdb/surrealdb/pull/5202/commits/464a9b1ba2fa3bd7b3af2069152983e40acb834d)